resources/datomic/schema.edn

```diff
@@ -268,6 +268,12 @@
   :db/doc         "The time that the game will start"}
 
  {:db/id          #db/id [:db.part/db]
+  :db/ident       :game/end-time
+  :db/valueType   :db.type/instant
+  :db/cardinality :db.cardinality/one
+  :db/doc         "The time that the game has ended"}
```

Comment:
I think we could but it may be a convoluted operation. (Pull in history api to query games last frame insert only if the game is in a closed state.) I will probably make a lot of adjustments around these time fields when I gain some more experience with the history api, but I feel keeping this field will be a little easier.